### PR TITLE
Command Line Emailer

### LINKED ISSUE #313: Command Line Emailer
## 🚀 Feature

The Python script will allow the user to send emails from their command line.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes, I have familiarized myself with the contributing guidelines.

## Motivation

I got the motivation while reading `Automate the Boring Stuff with Python` and realized we can save a lot of time of most of our day-to-day tasks can be automated.

## Pitch

This feature will be implemented with a Python script using Selenium webdriver

### PR DESCRIPTION
**Pull request by devgirlcodes**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

I have read the contributing guidelines

### Description

The python app automates the task of sending emails via the command line. It's especially useful if you have to send the same email to more than one recipient.

Fixes #313 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Review by photon149 — COMMENTED** [review]
@devgirlcodes Nice Project. Just a minor change required , add output SS in your README file.
No Duplicity ✅

**Comment by devgirlcodes** [comment]
> @devgirlcodes Nice Project. Just a minor change required , add output SS in your README file. No Duplicity ✅

I attached the screenshots in the readme. Could you have a look?

**Review by photon149 — APPROVED** [review]
Reviewed 👍🏻 
@Kushal997-das @prakhar3062 have a look

File: Desktop Application/Basic/Python/cmd-line-emailer/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+3. Turn On Javascript Enabled on your browser if you run into a problem while logging in
+
+## Output
+
+![login](https://user-images.githubusercontent.com/99732684/163577129-cae3f648-c388-47d1-a053-e72966752734.jpeg)
+
+The login window will be populated with the sender's email address and password already
+
+![message](https://user-images.githubusercontent.com/99732684/163577135-f4c9d4da-e3e5-4c67-a533-8ad792aeb51c.jpeg)
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/Python/cmd-line-emailer/readme.md:17]: Please reduce the width size of the all images .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@devgirlcodes Do the requires changes .

File: Desktop Application/Basic/Python/cmd-line-emailer/readme.md (hunk 1 above)
Comment (devgirlcodes) [reply Desktop Application/Basic/Python/cmd-line-emailer/readme.md]: > @devgirlcodes Do the requires changes .

@Kushal997-das I have resized the images.

**Comment by devgirlcodes** [comment]
> @devgirlcodes Do the requires changes .

I have made the requested changes.

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**